tests/src/Kernel/fixtures/markup_rendering_patterns/featured_media.yml

```diff
@@ -263,3 +263,37 @@ featured_media_iframe_with_text_container_right_aligned_with_link_ratio_21x9:
     contains:
       'div.col-12.col-md-6.order-md-2': 'Lorem Ipsum dolor sit amet.'
       'div.col-12.col-md-6.order-md-2 a': 'View all'
+featured_media_with_subtitle_and_subtitle_tag:
```

Comment:
use
```
  backward_compatibility:
    featured_media_subtitle_tag_h5: true
```
and list all situations with and without the the thing enabled, so you have a permutation of possibilities, in all cases the override takes priority and then you have h2 h3 or h5 depending on the BC value and weather title is set or not

Reply:
yeah